prompts/configs.go

```diff
@@ -267,10 +297,8 @@ func addPromptForField(key, defaultValue, validateRegex, validateMessage string,
 		input = input.Description(*description + "\n").Inline(false).Prompt("")
 	}
 
-	if !isQuickstart && defaultValue != "" {
+	if defaultValue != "" {
 		input = input.Value(&defaultValue)
```

Comment:
Not this is quite a big change @ryan-timothy-albert

Reply:
I think this is fine.